src/main/java/Ui.java

```diff
@@ -0,0 +1,45 @@
+import task.Task;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+
+    private Scanner sc;
+    private static final String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    private static final String GREETINGS = "Hello! I'm Duke\nWhat can I do for you?\n";
```

Comment:
I like how you declared the logo and welcome messages as constants instead of hard-coding them within the method! 👍